### ISSUE TITLE
Redesign for PhET-iO

### ISSUE THREAD

**Issue by zepumph** [phetsims/faradays-law#158]
Faradays law was worked on many years ago as the first "thoughtfully designed" PhET-iO sim that didn't have the "instrument everything" approach. We didn't quite take it to the finish line though. See #79. I think a new issue is warranted, though it may be worth looking at the end of that issue before we start this new process, for context. I also think this issue will co-opt #143, but with a more idiomatic name follows what we are doing in many repos. 

This is not a priority right now.
